src/components/Events.tsx

```diff
@@ -0,0 +1,35 @@
+import { useEffect, useState } from "react";
+import type { Event } from "../model/Event";
+import style from "./Events.module.scss";
+import React from "react";
+
+export const Events: React.FC = () => {
+  const [events, setEvents] = useState<Event[]>([]);
+
+  useEffect(() => {
+    const apiUrl = "https://sportujspolu-api.onrender.com/api/v1/events";
+
+    fetch(apiUrl)
+      .then((response) => {
+        if (!response.ok) {
+          throw new Error("Network response was not ok");
+        }
```

Comment:
I have seen this many times. I don't have a strong opinion about that cause I am unsure how often backend sends not-ok response that would NOT get caught otherwise by the try-catch. Maybe it is not needed